packages/api/package.json

```diff
@@ -41,10 +41,28 @@
     "@types/crypto-js": "4.1.0",
     "@types/jsonwebtoken": "8.5.8",
     "@types/md5": "2.3.1",
+    "@types/split2": "3.2.1",
     "aws-lambda": "1.0.7",
     "jest": "27.4.7",
+    "split2": "4.1.0",
     "supertokens-node": "8.6.0",
     "typescript": "4.5.5"
   },
-  "gitHead": "8be6a35c2dfd5aaeb12d55be4f0c77eefceb7762"
+  "gitHead": "8be6a35c2dfd5aaeb12d55be4f0c77eefceb7762",
+  "peerDependencies": {
+    "@clerk/clerk-sdk-node": "2.8.1",
```

Comment:
This'll mean that every install on a rw project will have a warning I think :/

Reply:
Even with the peerDependenciesMeta below saying this is optional?

I'm using this section of the yarn docs as my reference: https://yarnpkg.com/configuration/manifest#optionalDependencies

> If you need a package to be required only when a specific feature is used then use an optional peer dependency. Your users will have to satisfy it should they use the feature, but it won't cause the build errors to be silently swallowed when the feature is needed.